.circleci/config.yml

```diff
@@ -61,15 +61,25 @@ jobs:
       - run:
           name: Clone planet4-docker-compose
           command: |
-            git clone https://github.com/greenpeace/planet4-docker-compose
+            git clone --single-branch --branch planet-4998 https://github.com/greenpeace/planet4-docker-compose
```

Comment:
We have a bit of a chicken and egg problem: to switch this one to master again we need to merge https://github.com/greenpeace/planet4-docker-compose/pull/47/files, but that pipeline is failing because it depends on the changes from this PR.

Reply:
I suggest we merge that one first, then we can point this back to master before merging.